exercises/managing-dns/overview.md

```diff
@@ -36,3 +144,5 @@ Specify a nameserver to check using @
 - How do our local machines know which DNS server to communicate with?
 - How can we find out what DNS records exist on a domain?
 - How do you manage DNS entries as terraform resources?
```

Comment:
Probably, though maybe not. We can review the various providers on the docs page. The answer could be as simple as, "lookup your provider in the terraform docs"